src/schema/objects/enums.yaml

```diff
@@ -0,0 +1,1113 @@
+---
+# This file defines valid values in BIDS key-value pairs.
+_CoordUnits:
```

Comment:
Are these used for anything? Kept around for future use?

Reply:
I thought they were used elsewhere, but it looks like I was wrong. I'll just remove this group and any others that aren't actually used.